site/tutorials/snippets/08/manage/onboardService-crc.md

```diff
@@ -47,15 +47,15 @@ Duration: 5:00
 
 After onboarding the services, a built artifact of each service can be deployed.
 
-1. Deploy the carts-db service by executing the [keptn send event new-artifact](https://keptn.sh/docs/0.7.x/reference/cli/commands/keptn_send_event_new-artifact/) command:
+1. Deploy the carts-db service by executing the [keptn trigger delivery](https://keptn.sh/docs/0.8.x/reference/cli/commands/keptn_trigger_delivery/) command:
```

Comment:
This link is not yet working, I guess we still need to udpate docs. 
(**FYI: not a blocker, just for us to keep track**)